lib/screens/home/home.dart

```diff
@@ -0,0 +1,64 @@
+import 'package:flutter/material.dart';
+import 'package:ecellapp/screens/home/tabs/contactusScreen/contactusScreen.dart';
+import 'package:ecellapp/screens/home/tabs/menuScreen/menuScreen.dart';
+import 'package:ecellapp/screens/home/tabs/profileScreen/profileScreen.dart';
+import 'package:ecellapp/screens/home/widgets/selectedIconTheme.dart';
+
+class HomeScreen extends StatefulWidget {
+  @override
+  _HomeScreenState createState() => _HomeScreenState();
+}
+
+class _HomeScreenState extends State<HomeScreen> {
+  int _currentIndex = 1;
+  final List<Widget> _children = [ProfileScreen(), MenuScreen(), ContactusScreen()];
+  @override
+  Widget build(BuildContext context) {
+    double height = MediaQuery.of(context).size.height;
+    return Scaffold(
+      body: Stack(
+        children: <Widget>[
+          _children[_currentIndex],
+          Positioned(
+            left: 0,
+            right: 0,
+            bottom: 0,
+            child: _buildBottomNavBar(height),
+          ),
+        ],
+      ),
+    );
+  }
+
+  Widget _buildBottomNavBar(height) {
+    return ClipRRect(
+      borderRadius: BorderRadius.all(Radius.circular(height / 20)),
+      child: Container(
+          height: height / 10,
+          child: BottomNavigationBar(
+            currentIndex: _currentIndex,
+            onTap: onTabTapped,
+            selectedIconTheme: selectedIconTheme(), //TODO :Add Theme in the file under widgets
```

Comment:
there is no need to add a todo here coz
1. there is a todo in the file already, and
2. there are no changes w.r.t. this that is to be done in this file.

Reply:
done!